haproxy-operator/tests/integration/grpc_server/echo_pb2.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# ruff: noqa
+
+# -*- coding: utf-8 -*-
```

Comment:
Maybe we shouldn't have a license header in this file if we shouldn't edit it. Can you exclude it in the .licenserc.yaml configuration? You'll need to add one if it doesn't exist already